tweet-api/api/inference.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import urllib.request
+
+import numpy as np
+from scipy.special import softmax
+from transformers import AutoModelForSequenceClassification, AutoTokenizer
+
+task='emotion'
```

Comment:
Maybe run the code through `black` to get consistent formatting etc (@soulshake wdyt?)

Reply:
+1 for running Black (there should be a vsCode plugin @danasaur)